go.mod

```diff
@@ -60,8 +60,6 @@ require (
 
 replace mellium.im/sasl => github.com/mellium/sasl v0.2.1
 
-replace github.com/onflow/flow-go => ./
```

Comment:
Why did we have this in the first place?

Reply:
I remember there being a reason for this, but I can't remember why 😓 I'll try to dig up the PR.

I'd say that if everything builds and all the test pass without it, we should remove it.

Reply:
I think this was temporary during the GH org migration

Reply:
Found the relevant PR: https://github.com/dapperlabs/flow-go/pull/1959#discussion_r361214248

I think it's okay to remove all of these replaces if the tests pass without them

Reply:
I can't remove them all, as you can see, the build will failure. 

I can only remove this line to fix the build. But I don't understand why other "replace"s can't be removed

Reply:
I think part of the issue is the `integration` package is pinning a very old version of `flow-go`. With the `replace` that old version gets replaced by whatever is in the replacement path on the machine but without the replace we actually fetch that old version.